src/components/FullWidthPromo/FullWidthPromo.tsx

```diff
@@ -15,8 +15,7 @@ type FullWidthPromoProps = {
   imageCredit?: string;
   imageHeight?: string;
   imageLicence?: string;
-  imageSizes?: string;
```

Comment:
Same comment as above re `sizes` attribute, shouldn't it be kept for srcSet usage?